mathesar_ui/src/sections/table-view/header/Header.svelte

```diff
@@ -75,8 +84,12 @@
 
   {#each $columnsDataStore.columns as column (column.name)}
     <HeaderCell {column} {meta}
-      columnPosition={getColumnPosition($columnPositionMap, column.name)}/>
+      columnPosition={getColumnPosition($columnPositionMap, column.name)} on:columnDelete={openColumnModal(column)}/>
```

Comment:
@pavish @seancolsen Just a heads up, I'm seeing a TypeScript warning on `{openColumnModal(column)}` on VS. No linting error.

Reply:
Yeah that's because we need to pass a function into the event. You're _calling_ the function instead of passing it in. I fixed this in 6944f8a9725edbeb6d1cddae5d5cdb804ae23a5d